README.md

```diff
@@ -37,10 +37,10 @@ Features include
 * [Made for 99% developers](https://future.a16z.com/software-development-building-for-99-developers/)
 * [High-quality API documentation](https://web3-ethereum-defi.readthedocs.io/)
 * [Fully type hinted](https://web3-ethereum-defi.readthedocs.io/) for good developer experience
-* [ERC-20 token issuance and manipulation](https://web3-ethereum-defi.readthedocs.io/en/latest/_autosummary/eth_hentai.token.html#module-eth_hentai.token)
+* [ERC-20 token issuance and manipulation](https://web3-ethereum-defi.readthedocs.io/en/latest/_autosummary/eth_defi.token.html#module-eth_defi.token)
```

Comment:
@miohtama `/en/latest/....` URLs don't seem to work anymore for some reason, is it some config in RTD?